packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/index.tsx

```diff
@@ -1,16 +1,22 @@
 import React from "react";
 
 import { Wizard } from "../common/Wizard";
+import { PublicodeProvider } from "../publicodes";
 import { initialState, stepReducer } from "./stepReducer";
 
-const SimulateurPreavisRetraite = ({ icon, title }) => (
-  <Wizard
-    icon={icon}
-    title={title}
-    initialState={initialState}
-    // @ts-ignore
-    stepReducer={stepReducer}
-  />
+const SimulateurPreavisRetraite = ({ icon, title, publicodes }) => (
+  <PublicodeProvider
+    publicodes={publicodes}
+    rule="contrat salarié . préavis de retraite"
```

Comment:
and this one with something like `goal` `endRule`, `resultRule`, `aimRule`, `targetRule`

Reply:
IMO:

* goal : too generic
* endRule: too vague, what is a end ?
* resultRule: A good one
* aimRule: aim it's more for gun, otherwise it should `aimAtRule` / `aimForRule`. Not really fan.
* targetRule: Seems the better option for me :) 

I choose `targetRule` :)

Reply:
thanks